mcs/build/tests.make

```diff
@@ -22,7 +22,7 @@ ifndef NO_TEST
 test_nunit_lib = nunitlite.dll
 xunit_core := xunit.core xunit.abstractions xunit.assert Xunit.NetCore.Extensions
 xunit_deps := System.Runtime
-xunit_src  := $(patsubst %,$(topdir)/../external/xunit-binaries/%,BenchmarkAttribute.cs BenchmarkDiscover.cs)
+xunit_src  := $(patsubst %,$(topdir)/../external/xunit-binaries/%,BenchmarkAttribute.cs BenchmarkDiscover.cs) $(topdir)/../mcs/class/test-helpers/PlatformDetection.cs
```

Comment:
why the stub instead of `../../../external/corefx/src/Common/tests/System/PlatformDetection.cs` ?

Reply:
because original is hundreds lines long in 3 files where we need a single property